CMakeLists.txt

```diff
@@ -177,7 +177,7 @@ if (WIN32)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
     endif()
 endif()
-if (CMAKE_SYSTEM_NAME MATCHES "CYGWIN")
+if (CYGWIN OR MINGW)
```

Comment:
isn't in this test the `CMAKE_SYSTEM_NAME MATCHES` part missing twice?

Reply:
I don't understand your question

Reply:
Shouldn't the test be something like `if (CMAKE_SYSTEM_NAME MATCHES "CYGWIN" OR CMAKE_SYSTEM_NAME MATCHES "MINGW")`?